resources/docs/usage.rst

```diff
@@ -28,7 +28,7 @@ You can use the `Query Builder`_:
 
 .. caution::
 
-    Don't forget to `enable the DB aliases`_ if you want to use the ``DB``
+    Don't forget to [enable the DB aliases](configuration.rst) if you want to use the ``DB``
```

Comment:
you can't use markdown in rst. Does this work: `` `enable the DB aliases <configuration.rst>`_``?